mkinitcpio

```diff
@@ -211,25 +211,33 @@ build_image() {
             ;;
     esac
 
-    cpio_opts=('-0' '-o' '-H' 'newc')
-    (( _optquiet )) && cpio_opts+=('--quiet')
+    cpio_opts=('--null' '-cf-' '--format=newc')
+    (( _optquiet )) || cpio_opts+=('-v')
```

Comment:
Let's not do this -- just drop the opportunity to add -v entirely. At best, it'll just be nonsense on stderr. at worst, i'm not even sure... I would seriously doubt that it'll write to stdout, but this doesn't seem like nice default behavior.

Reply:
Sure.